Dockerfile

```diff
@@ -10,17 +10,15 @@ RUN apt-get update \
     libpng-dev libtiff-dev pandoc git libgit2-dev \
     && rm -rf /var/lib/apt/lists/*
 
-RUN Rscript -e 'install.packages(c("pak", "renv"))'
+RUN Rscript -e 'install.packages("pak")'
 
 COPY . /workflow.data.preparation
```

Comment:
NIT: I recommend copying in `DESCRIPTION` first, installing the `deps` there, and then copying in the rest of the repository. 

That takes advantage of `Dockerfile` caching and ensuring faster builds.

Reply:
opened #109